android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/Utils.java

```diff
@@ -58,6 +58,8 @@
   private static final String PWSTRIPTIME_KEY = "pwstriptime";
   private static final RegionResolver REGION_RESOLVER = new RegionResolver();
 
+  private static boolean userHasBeenWarned = false;
```

Comment:
Is there somewhere else we could store this bit?  We should try not to store application state in the Utils class.